sdk/tables/azure-data-tables/azure/data/tables/_serialize.py

```diff
@@ -4,21 +4,20 @@
 # license information.
 # --------------------------------------------------------------------------
 # pylint: disable=no-self-use
-import sys
-import uuid
 from uuid import UUID
 from datetime import datetime
-
-from math import (
-    isnan,
-)
+from math import isnan
+import sys
+import uuid
+import isodate
 
 from azure.core import MatchConditions
+from azure.core.exceptions import raise_with_traceback
+
 from ._entity import EdmType, EntityProperty
 from ._models import TablePayloadFormat
-from ._shared._common_conversion import _to_str, _encode_base64, _to_utc_datetime
-from ._shared._error import _ERROR_VALUE_TOO_LARGE, _ERROR_TYPE_NOT_SUPPORTED
-
+from ._common_conversion import _to_str, _encode_base64, _to_utc_datetime
+from ._error import _ERROR_VALUE_TOO_LARGE, _ERROR_TYPE_NOT_SUPPORTED
 
 
 _SUPPORTED_API_VERSIONS = [
```

Comment:
i had mentioned this in a previous comment (about adding an `ApiVersion` class), but you can have these two API versions in that class

Reply:
There is an issue out for correcting the issues surrounding the API version. I think it's better to address version issues all in one PR.
#12411 and #12783